cypress/integration/admin/addingATrust.js

```diff
@@ -50,6 +50,7 @@ describe("As an admin, I want to add a trust so that a trust can use the virtual
     cy.get("input[name=trust-admin-code]").type("bowcode");
     cy.get("input[name=trust-password]").type("bowpassword");
     cy.get("input[name=trust-password-confirmation]").type("bowpassword");
+    cy.get("select").select("whereby");
```

Comment:
Could we be more explicit here with the `cy.get`? E.g. `cy.get("select[name=video-provider]` so if a newcomer came along they'd know Whereby is a video provider and also if we added another dropdown we don't have to update this.

Reply:
Good point. I was being a bit lazy with the select here, will update thanks